### PR TITLE
Glt 820 pcsi 2

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: src/main/java/ca/on/oicr/gsi/migration/converters/SampleAttribute.java [hunk 1]
```diff
@@ -574,10 +584,10 @@ public boolean convert(MigrationSample sample, String value) {
       if (sample.getLab() == null) sample.setLab(new LabImpl());
       if (sample.getLab().getAlias() == null) sample.getLab().setAlias("Not Specified");
       if (sample.getLab().getInstitute() == null) sample.getLab().setInstitute(new InstituteImpl());
-      if (sample.getLab().getInstitute().getAlias() != null && !sample.getLab().getInstitute().getAlias().equals(value)) {
+      if (sample.getLab().getInstitute().getAlias() != null && !sample.getLab().getInstitute().getAlias().equals(value.trim())) {
         return false;
       }
-      sample.getLab().getInstitute().setAlias(value);
+      sample.getLab().getInstitute().setAlias(value.trim());
```
Comment (morgantaschuk) [inline src/main/java/ca/on/oicr/gsi/migration/converters/SampleAttribute.java]: Arguably, trimming should be done already before the value is passed to convert. Same thing on line 587.

File: src/main/java/ca/on/oicr/gsi/migration/converters/SampleAttribute.java [hunk 2]
```diff
@@ -941,19 +954,19 @@ public boolean convert(MigrationSample sample, String value) {
...
-  
+
   private static final Map<String, SampleAttribute> lookup = new HashMap<>();
   static {
     for (SampleAttribute att : EnumSet.allOf(SampleAttribute.class)) {
       lookup.put(att.getKey(), att);
     }
   }
-  
+
```
Comment (morgantaschuk) [inline src/main/java/ca/on/oicr/gsi/migration/converters/SampleAttribute.java]: It's happening to you too.

File: src/main/java/ca/on/oicr/gsi/migration/converters/SampleAttribute.java (hunk 1 above)
Comment (djcooke) [inline src/main/java/ca/on/oicr/gsi/migration/converters/SampleAttribute.java]: Agreed. Changed AbstractSampleConverter to trim all String values (see new commit)

**Comment by morgantaschuk** [comment]
:+1:

**Comment by apmasell** [comment]
Trim *ALL* the things. :+1:

**Pull request merged**